src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-sync-pipe-modal/rgw-multisite-sync-pipe-modal.component.ts

```diff
@@ -105,6 +116,14 @@ export class RgwMultisiteSyncPipeModalComponent implements OnInit {
       });
   }
 
+  replaceWithAsterisk(zones: string[]) {
+    return zones.map((str) => str.replace(ALL_ZONES, '*'));
+  }
+
+  replaceAsteriskWithConst(zones: string[]) {
```

Comment:
nit: since we are using strings
```suggestion
  replaceAsteriskWithString(zones: string[]) {
```